daprdocs/content/en/developing-applications/building-blocks/workflow/workflow-overview.md

```diff
@@ -0,0 +1,49 @@
+---
+type: docs
+title: Workflow overview
+linkTitle: Overview
+weight: 1000
+description: "Overview of the workflow building block"
+---
+
+With Dapr workflows, you can automate and orchestrate tasks within your application. The workflow feature provides a developer-friendly programming model for authoring workflows as code in a way that abstracts away the complexities of messaging, state management, and failure handling.
+
+The workflow feature is comprised of the Workflow API and the workflow components that allow for authoring workflows naitively in Dapr or through external 3rd party workflow applications.
+
+<img src="/images/concepts-building-blocks.png" width=250>
+
+## Workflow API
+
```

Comment:
The workflow building block:
- Provides a workflow API for running workflows 
- An built-in workflow runtime to write Dapr workflows (of type `workflow.dapr`)
- Integrates with various workflow runtimes as components, for example Temporal workflows.